python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -114,7 +115,11 @@ def ggml_convert_quant(model, qtype, optimize_model=True, device="cpu"):
         pass
 
     if optimize_model:
-        model = optimize(model)
+        try:
+            model = optimize(model)
+        except:
+            logger.info("It is not supported that optimizing a model isn't belong to"
```

Comment:
what do you mean?

Reply:
`optimize()`  is only compatible with the Hugging Face API, and take effects on `chatglm` and `llama` family more specifically.
We apply `optimize()` on other pytorch model is likely to fail, `optimize()` strongly assume it is [a hf format model](https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/transformers/convert.py#L142):
```python
if "chatglm2" in model.config._name_or_path:
```

Do we need to explore a way more general to extend its working range?

Reply:
I see

Reply:
If there is any issue, you should fix `optimize` instead